source/sass/patterns/organisms/site-header.scss

```diff
@@ -23,72 +20,65 @@ $siteHeader-block_end_space: 7px !default;
   @include max-inline-size($grid-max_width);
 
   position: relative;
-  @supports (display:flex) {
+  @supports (display: flex) {
     position: static;
     display: flex;
     align-items: center;
   }
 
   --block-size: #{$siteHeader-block_size__default};
+  --inline-space: #{$siteHeader-inline-space__default};
   @include block-size($siteHeader-block_size__default);
+  @include padding($siteHeader-inline-space__default, inline);
 
   @include mq($from: medium) {
     --block-size: #{$siteHeader-block_size__medium};
+    --inline-space: #{$siteHeader-inline-space__medium};
     @include block-size($siteHeader-block_size__medium);
+    @include padding($siteHeader-inline-space__medium, inline);
+  }
+
+  @include mq($from: x-wide) {
+    --inline-space: 0;
+    @include padding(0, inline);
   }
 
   @supports (--name: value) {
     @include block-size(var(--block-size));
+    @include padding(var(--inline-space), inline);
   }
 }
 
 .site-header__logo_link {
 
+  position: relative;
```

Comment:
Probably needs a `max-inline-size` here to cater to SVGs without specified internal `width` and `height` attributes.